src/main/resources/config/application.yml

```diff
@@ -9,11 +9,11 @@ spring:
       fail-on-unknown-properties: true
  # Keycloak Server Configurations
 keycloak:
-  realm: ${KEYCLOAK_REALM_NAME:saas-realm}
+  realm: ravi_realm
```

Comment:
please update these details here as suggested by Karthik, and in future, please refrain from pushing this file unless required